pkgs/development/libraries/openimagedenoise/default.nix

```
@@ -3,15 +3,20 @@
   config,
   cudaPackages,
   cudaSupport ? config.cudaSupport,
+  darwin,
   fetchzip,
   ispc,
   lib,
   python3,
   stdenv,
   tbb,
+  xcodebuild,
 }:
 
-stdenv.mkDerivation rec {
+let
+  stdenv' = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
+in
+stdenv'.mkDerivation rec {
```

Nixpkgs seems to move away from using `rec` towards `finalAttrs`.
☝️ If this suggestion is accepted, please ensure that the opening parenthesis before `finalAttrs` is closed at the end of the file (for some odd reason I cannot make that suggestion here currently…)
```suggestion
stdenv'.mkDerivation (finalAttrs: {
```